components/backup/base/dpa.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: velero-aws
+spec:
+  backupLocations:
+    - velero:
+        config:
+          profile: default
+          region: us-east-1
```

Comment:
Here I need to add `kmsKeyId: kmsKeyId`, do we have it in vault so I can get the value from a secret I will create through ExternalSecret. @jrosental ?

Reply:
Done